### ISSUE TITLE
[Bug] r.in.nasadem error importing local nasadem tiles

### ISSUE THREAD

**Issue by rcammi** [OSGeo/grass-addons#861]
**Name of the addon**
r.in.nasadem

**Describe the bug**
When importing local nasadem tiles fails with this error:

```
Importing 1 NASADEM tiles...
 100%

ERROR: At least one of the following options is required: <pattern> and <name>
Traceback (most recent call last):
  File "/root/.grass8/addons/scripts/r.in.nasadem", line 647, in <module>
    main()
  File "/root/.grass8/addons/scripts/r.in.nasadem", line 567, in main
    grass.run_command(
  File "/usr/local/grass/etc/python/grass/script/core.py", line 541, in run_command
    return handle_errors(returncode, result=None, args=args, kwargs=kwargs)
  File "/usr/local/grass/etc/python/grass/script/core.py", line 429, in handle_errors
    raise CalledModuleError(module=module, code=code, returncode=returncode)
grass.exceptions.CalledModuleError: Module run `g.remove --q -f type=raster name=` ended with an error.
The subprocess ended with a non-zero return code: 1. See errors above the traceback or in the error output.
```

**To Reproduce**

1. Install r.in.nasadem addon in grass
2. Run grass and set region of mapset
3. Download to local folder /grassdb/dem/ nasadem tiles
3. Execute  
r.in.nasadem version="NASADEM_HGT.001" layer="hgt" local="/grassdb/dem/" output="nasadem_test" memory=2000

**System description (please complete the following information):**
Operating System: Ubuntu 20.04.5 LTS
GRASS GIS version: GRASS 8.2.0 (2022)

**Comment by neteler** [comment]
How did you solve it, @rcammi ?

**Comment by rcammi** [comment]
Hi! It seems someone tried to fix it months ago: https://github.com/OSGeo/grass-addons/commit/df511d629a3e4eb0738131fff6b9179adbad9943

I deleted the addon and reinstalled it in grass and It worked. I believe when I first tried the new updates were not available yet.

**Comment by neteler** [comment]
> Hi! It seems someone tried to fix it months ago: [df511d6](https://github.com/OSGeo/grass-addons/commit/df511d629a3e4eb0738131fff6b9179adbad9943)

Glad the fix by Guido does it!

(currently we have no mechanism in place to see if one is using an outdated addon...)
